src/cluster_legacy.c

```diff
@@ -6842,6 +6842,23 @@ int clusterCommandSpecial(client *c) {
     } else if (!strcasecmp(c->argv[1]->ptr, "links") && c->argc == 2) {
         /* CLUSTER LINKS */
         addReplyClusterLinksDescription(c);
+    } else if (!strcasecmp(c->argv[1]->ptr, "flushslot") && (c->argc == 3 || c->argc == 4)) {
+        /* CLUSTER FLUSHSLOT <slot> [ASYNC|SYNC] */
+        int slot;
+        int lazy = 1;
```

Comment:
Let's use `server.lazyfree_lazy_user_flush` by default? That's the default for FLUSHALL when the caller doesn't specify sync or async.

Reply:
fixed.